research-institute-for-nature-and-forest.csl

```diff
@@ -0,0 +1,267 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never">
+  <info>
+    <title>Research Institute for Nature and Forest (Instituut voor Natuur- en Bosonderzoek)</title>
+    <title-short>INBO</title-short>
+    <id>http://www.zotero.org/styles/research-institute-for-nature-and-forest</id>
+    <link href="http://www.zotero.org/styles/research-institute-for-nature-and-forest" rel="self"/>
+    <author>
+      <name>Maarten Stevens</name>
+      <uri>http://www.mendeley.com/profiles/maarten-stevens/</uri>
+    </author>
+    <contributor>
+      <name>Thierry Onkelinx</name>
+      <uri>http://www.mendeley.com/profiles/thierry-onkelinx/</uri>
+    </contributor>
+    <contributor>
+      <name>Floris Vanderhaeghe</name>
+      <email>floris.vanderhaeghe@inbo.be</email>
+    </contributor>
+    <category citation-format="author-date"/>
+    <category field="biology"/>
+    <updated>2019-08-01T00:00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en">
+    <terms>
+      <term name="in">In</term>
+    </terms>
+  </locale>
+  <locale xml:lang="nl">
+    <terms>
+      <term name="in">In</term>
+      <term name="et-al">et al.</term>
+    </terms>
+  </locale>
+  <macro name="container">
+    <choose>
+      <if type="chapter paper-conference" match="any">
+        <text term="in" prefix=". " suffix=": "/>
+        <names variable="editor translator" delimiter=", " suffix=". ">
+          <name delimiter-precedes-last="always" initialize-with="." name-as-sort-order="all" sort-separator=" "/>
+          <label text-case="lowercase" prefix=" (" suffix=")"/>
+        </names>
+        <group delimiter=", ">
+          <text variable="container-title"/>
+          <text variable="collection-title"/>
+        </group>
+      </if>
+      <else-if type="bill book graphic legal_case legislation motion_picture report song" match="any">
+        <group prefix=". " delimiter=", ">
+          <text variable="container-title"/>
+          <text variable="collection-title"/>
+        </group>
+      </else-if>
+      <else>
+        <group prefix=". " delimiter=", ">
+          <text variable="container-title"/>
+          <text variable="collection-title"/>
+        </group>
+      </else>
+    </choose>
+  </macro>
+  <macro name="author">
+    <names variable="author">
+      <name and="symbol" delimiter-precedes-last="never" et-al-min="10" et-al-use-first="10" initialize-with="." name-as-sort-order="all" sort-separator=" "/>
```

Comment:
`et-al-min="10" et-al-use-first="10"`

@florisvdh, `et-al-min` always needs to have a higher value than `et-al-use-first`. Per http://docs.citationstyles.org/en/1.0.1/specification.html#name:

> If the number of names in a name variable matches or exceeds the number set on et-al-min, the rendered name list is truncated after reaching the number of names set on et-al-use-first.